frontend/src/scenes/ingestion/Sidebar.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import posthogLogo from 'public/posthog-logo.png'
+import { ingestionLogic } from './ingestionLogic'
+import { useActions, useValues } from 'kea'
+import './IngestionWizard.scss'
+import { InviteMembersButton, SitePopover } from '~/layout/navigation/TopBar/SitePopover'
+import { eventUsageLogic } from 'lib/utils/eventUsageLogic'
+import { LemonButton } from 'lib/components/LemonButton'
+import { IconArticle, IconQuestionAnswer } from 'lib/components/icons'
+import { HelpType } from '~/types'
+import { LemonDivider } from 'lib/components/LemonDivider'
+
+const HELP_UTM_TAGS = '?utm_medium=in-product-onboarding&utm_campaign=help-button-sidebar'
+
+export function Sidebar(): JSX.Element {
+    const { currentIndex, platform } = useValues(ingestionLogic)
+    const { setVerify, setPlatform } = useActions(ingestionLogic)
+    const { reportIngestionHelpClicked } = useActions(eventUsageLogic)
+
+    return (
+        <div className="IngestionSidebar">
+            <div className="IngestionSidebar__content">
+                <div style={{ paddingLeft: 8 }}>
+                    <img src={posthogLogo} style={{ width: 157, height: 30 }} />
+                </div>
+                <div className="IngestionSidebar__steps">
+                    <b>
+                        <div
+                            className={`mb ${currentIndex === 0 && 'ingestion-current-nav-step'}`}
+                            onClick={() => setPlatform(null)}
+                        >
+                            <span>Get started</span>
+                        </div>
+                        <div
+                            className={`mb ${currentIndex === 1 && 'ingestion-current-nav-step'} ${
+                                !platform && 'nonclickable'
+                            }`}
+                            onClick={() => setVerify(false)}
+                        >
+                            <span>Connect your product</span>
+                        </div>
+                        <div
+                            className={`mb ${currentIndex === 2 && 'ingestion-current-nav-step'} ${
+                                !platform && 'nonclickable'
+                            }`}
+                            onClick={() => {
+                                if (platform) {
+                                    setVerify(true)
```

Comment:
There's some odd behavior when users click these buttons. For example, if you're on 'get started' and you click 'connect your product' it jumps to 'listen for events'. Then if you click 'get started' it moves to 'connect your product.

Because we require information from the previous step to inform what the next step shows, I wonder if these should not be clickable when moving forward?

Reply:
Oooo just seeing the `nonclickable` class, so I'm guessing this is just a bug. Also, it might makes sense to use `LemonButton` here instead of a click-able div.

Reply:
I think that might've been the only bug with the nav steps? And yeah I'll replace these with lemon buttons for consistency